.prow.sh

```diff
@@ -4,7 +4,10 @@
 
 # # Only these tests make sense for csi-proxy
 : ${CSI_PROW_TESTS:="unit"}
+: ${CSI_PROW_BUILD_PLATFORMS:="linux amd64; windows amd64 .exe"}
```

Comment:
I thought linux build does not work?

Reply:
somehow I saw it passed before, but now it always fail. 
I removed the linux part.